frontend/src/component/project/Project/ProjectSettings/ChangeRequestConfiguration/ChangeRequestConfiguration.tsx

```diff
@@ -82,29 +130,59 @@ export const ChangeRequestConfiguration: VFC = () => {
                 disableGlobalFilter: true,
                 disableSortBy: true,
             },
+            {
+                Header: 'Required approvals',
+                align: 'center',
+                Cell: ({ row: { original } }: any) => (
+                    <ConditionallyRender
+                        condition={original.changeRequestEnabled}
+                        show={
+                            <StyledBox data-loading>
+                                <GeneralSelect
+                                    options={approvalOptions}
+                                    value={original.requiredApprovals || 1}
+                                    onChange={approvals => {
```

Comment:
can be extracted to a separate function because the code is getting very dense here